src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -374,7 +374,7 @@ module Tick : S = struct
                                + double (cell (var (Witness 4, Curr))) ) ) ) )
                )
              + alpha_pow 1
-               * ( (cell (var (Witness 2, Next)) * cell (var (Witness 2, Next)))
+               * ( square (cell (var (Witness 2, Next)))
```

Comment:
You probably need to checkout the version of `src/lib/crypto/proof-systems` from develop and rebuild. These changes are from an older commit.